test/membrane_ice_plugin/integration_test.exs

```diff
@@ -55,7 +60,7 @@ defmodule Membrane.ICE.IntegrationTest do
     ]
 
     msg = {:connectivity_check, binding_request, self()}
-    Testing.Pipeline.message_child(pid, :ice, msg)
+    Testing.Pipeline.message_child(pipeline, :ice, msg)
```

Comment:
When you will change eg. `handle_parent_notification({:connectivity_check, request, alloc_pid}, _ctx, state)` on `handle_info`, this line will stop work. To fix this, you have to send this message not via action returned from `Testing.Pipeline`, but via explicit `send/2`